.github/workflows/test.yaml

```diff
@@ -0,0 +1,25 @@
+name: test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  build:
+    name: go
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
I guess this doesn't matter all that much since there's only one thing in the matrix, but I was curious why not just let this default to true?

Reply:
I have no idea, I just copied this file from the upstream and changed it to use 1.16 instead of 1.18 and 1.19. I tried just adding 1.16, but apparently gofmt behaves differently in the new versions.